packages/components/src/components/tag/_tag.scss

```diff
@@ -108,28 +106,34 @@
     }
   }
 
-  .#{$prefix}--tag--filter > svg {
+  .#{$prefix}--tag__close-icon {
     flex-shrink: 0;
     width: rem(20px);
     height: rem(20px);
     margin: 0 0 0 rem(4px);
     padding: rem(2px);
     border: 0;
-    fill: $inverse-01;
     background-color: transparent;
     border-radius: 50%;
+    cursor: pointer;
 
     &:hover {
       background-color: $inverse-hover-ui;
     }
   }
 
-  .#{$prefix}--tag--filter:focus > svg {
+  .#{$prefix}--tag__close-icon svg {
+    fill: $inverse-01;
```

Comment:
For my own edification, why did you pull this out? I see it done a lot throughout our styles and I wanted to learn why, thanks!

Reply:
are you referring to the selector? `.#{$prefix}--tag--filter` is no longer a `button` and won't receive focus by default, and the icon is now contained in `button.bx--tag__close-icon`